cirq/interop/quirk/url_to_circuit_test.py

```diff
@@ -79,15 +79,15 @@ def test_parse_with_qubits():
 
     assert quirk_url_to_circuit(
         'http://algassert.com/quirk#circuit={"cols":[["H"],["•","X"]]}',
-        qubits=cirq.GridQubit.rect(4, 4)) == cirq.Circuit(
-            cirq.H(a),
-            cirq.X(b).controlled_by(a))
+        qubits=cirq.GridQubit.rect(4, 4),
+    ) == cirq.Circuit(cirq.H(a),
```

Comment:
It's fairly common to add commas in long literal lists/tuples/sets/etc, but I suppose people are likely to forget to do this in function argument lists. From a quick glance at https://github.com/psf/black/issues/1629, it seems that the fallout is fairly scary: black produces inequivalent code! However, they do have a workaround (running twice with --fast), so perhaps we should use that in #3516? Added a comment there.